llvm/lib/Transforms/IPO/EmbedBitcodePass.cpp

```diff
@@ -41,5 +41,5 @@ PreservedAnalyses EmbedBitcodePass::run(Module &M, ModuleAnalysisManager &AM) {
 
   embedBufferInModule(M, MemoryBufferRef(Data, "ModuleData"), ".llvm.lto");
 
-  return PreservedAnalyses::all();
+  return PreservedAnalyses::none();
```

Comment:
I guess, technically we could predicate none/all on if its ThinLTO, but that also depends on internal details of the different BitcodeWritters.